products/kubic/main.pm

```diff
@@ -67,10 +79,15 @@ if (get_var 'STACK_ROLE') {
 else {
     loadtests 'boot_from_dvd';
     if (get_var 'SYSTEM_ROLE') {
-        loadtests 'installation';
-        loadtests 'boot_from_disk';
-        loadtests 'feature' if (check_var 'EXTRA', 'FEATURES');
-        loadtest 'shutdown/shutdown';
+        if (check_var('HDDSIZEGB', '10')) {
```

Comment:
Is it the case, that bug occurs only when have SYSTEM_ROLE?

Reply:
Actually, because they have handle it in its own "openQA product: kubic". This is already including every is_caasp scenario. There is no other way of changing the schedule for Kubic, without making a big refactoring of this code. So, the only condition here is when disk is smaller than supported.

Reply:
we can use `is_caasp 'kubic'` to check if it's kubic, but I was more asking if we can move that condition out of `SYSTEM_ROLE` condition. Do all kubic installations include system roles?

Reply:
Which is not installation. But fine if we detect it like this.